firmware/1.5.4.1/app/include/user_config.h

```diff
@@ -97,7 +97,7 @@ extern void luaL_assertfail(const char *file, int line, const char *message);
 #define LUA_PROCESS_LINE_SIG 2
 #define LUA_OPTIMIZE_DEBUG      3
 
-#define ENDUSER_SETUP_AP_SSID "AlarmPanel"
+#define ENDUSER_SETUP_AP_SSID "konnected-io"
```

Comment:
I was thinking the discovered list in SmartApp should be consistent with the SSID. Previosly it was AlarmPanel_XXXXXX in both SSID and SmartApp, so if we change the SSID to just konnected-io, we won't get both places to be consistent.